capture/devconfig.py

```diff
@@ -74,7 +80,9 @@
             'reagent_alias': 'Precursor',
             'required_files': ['observation_interface', 'preparation_interface', 'metadata.json'],
             'required_folders': [],
-            'observation_interface': {'uid_col': 'F'}
+            'observation_interface': {'uid_col': 'F',
```

Comment:
Didn't MIT want us to change the location of the UIDs to like, B, or something similar.

Reply:
Trivial change and I think you are right.  Updated.